pkg/subctl/cmd/gather/connectivity.go

```diff
@@ -0,0 +1,120 @@
+/*
```

Comment:
@skitt Does `subctl/cmd/gather/*` make sense or you think it should be `subctl/gather/*`

Reply:
subctl/cmd/gather sounds fine to me

Reply:
I vote for `subctl/cmd/gather` too.